src/examples/pongxr/scene.js

```diff
@@ -0,0 +1,200 @@
+    const level = new Level(curPosRot);
+    scene.add(level);
+    networking.remoteSync.addLocalObject(level, { type: "level", posRot: curPosRot }, true);
+
+    // dumb hack for the other side 
+    placementCube.scale.set(0, 0, 0);
+
+    // local
+    scene.remove(placementCube);
+    // remote placement cube removal, not currently working on metachromium
+    networking.remoteSync.removeSharedObject(3);
+
+    // BALL
+    if (State.isMaster)
+    {
+        ball = new Ball(targetPosition, true);
+        ball.initPos = targetPosition;
+        scene.add(ball);
+        networking.remoteSync.addLocalObject(ball, { type: "ball", position: targetPosition }, true);
+    }
+}
+
+const initPlacement = () =>
+{
+    State.GameState = GameState.placement;
+    placementCube = PlacementCube();
+    scene.add(placementCube);
+    networking.remoteSync.addSharedObject(placementCube, 3, true);
+}
+
+/// GAME STATE
+
+State.eventHandler.addEventListener("gameover", (e) =>
+{
+    if (ball != undefined)
+    {
+        ball.reset();
+    } else
+    {
+        console.error("can't reset; no ball found!");
+    }
+});
+
+/// INPUT
+
+let doubleClick = false;
+
+State.eventHandler.addEventListener("select", e =>
+{
+    switch (State.GameState)
+    {
+        case (GameState.placement):
+            if (State.isMaster)
+            {
+                createPongLevel(placementCube);
+            }
+            break;
+
+        case (GameState.play):
+        default:
+            if (!doubleClick)
+            {
+                doubleClick = true;
+                setTimeout(function ()
+                {
+                    if (ball == undefined)
+                    {
+                        console.error("can't kickoff; no ball found!");
+                        return;
+                    }
+                    ball.kickoff();
+                    doubleClick = false;
+                }, 200);
+            }
+            else
+            {
+                if (State.debugMode) console.log("doubleclick");
+                // TOOD: implement dclick reset
+                // location.reload();
+            }
+            break;
+    }
+});
+
+// Add paddles when we know our inputs
+State.eventHandler.addEventListener("inputsourceschange", e =>
+{
+    if (paddles.length != 0) return //  avoid false positives, i.e headset put down, but paddles already instantiated
+
+    XRInput.controllerGrips.forEach((e, i) =>
+    {
+        const paddle = new Paddle();
+        scene.add(paddle);
+        paddles.push(paddle);
+        networking.remoteSync.addLocalObject(paddle, { type: "paddle" }, true);
+    });
+
+    // local paddle controller component to control player's networked paddle
+    // note: *all* XRInput data is local. 
+    // only the objects sync'd to it are networked.
+    // note2: we don't add it to rAF via Update() due to accumulated rAF lagginess over time.
+
+    setInterval(function ()
```

Comment:
How come you chose to put this in an interval loop and not in an `Update` loop?

Reply:
ah yes, there's the rub - I'm hinting at this in line 139. I'm still not 100% sure why but polling the controllers on rAF is what caused the staggered latency in the networked experiences. I think it's because we're coercing a much higher polling rate than the WebRTC packet transmission rate and the data piles up over time. 

It felt cringey to do when we have such a nice Update() pattern but, specifically in the case of networked, physics-enabled controllers, I'm not 100% sure what the cleaner solve is.

Reply:
What if you passed state to RemoteSync when you registered the object and use that state to throttle updates in RemoteSync.sync or RemoteSync.checkUpdate?

Reply:
I have a feeling most SandCastle users are going to run into this pretty quickly.

Reply:
definitely. I'm hesitant to throttle _all_ networking (placementCube is a good example of something that doesn't appear to need it) but agree that this is going to recur frequently. Looking forward to talking about it.

Reply:
I'll open an issue